main/config/settings/.defaults.yml

```diff
@@ -229,3 +229,11 @@ dbStudyPermissions: ${self:custom.settings.dbPrefix}-StudyPermissions
 
 # DynamoDB table name for Environments created using AWS Service Catalog
 dbEnvironmentsSc: ${self:custom.settings.dbPrefix}-EnvironmentsSc
+
+# Arn partition, 'aws' for global regions, 'aws-cn' for China Mainland regions
+regionPartition: aws
+
+# AWS domain 
+# China Mainland regions: 'amazonaws.com.cn'
+# Global regions: 'amazonaws.com'
+awsDomain: amazonaws.com
```

Comment:
get the suffix from sdk

Reply:
This suffix is also be used in serverless framework which does not have aws domain suffix config for both global regions and china regions.